shell/browser/native_window_views.cc

```diff
@@ -668,6 +668,22 @@ bool NativeWindowViews::IsResizable() {
   return CanResize();
 }
 
+void NativeWindowViews::SetAspectRatio(double aspect_ratio,
+                                       const gfx::Size& extra_size) {
+#if defined(OS_LINUX)
+  NativeWindow::SetAspectRatio(aspect_ratio, extra_size);
+  gfx::SizeF aspect(aspect_ratio, 1.0);
+  // widget SetAspectRatio() won't work with numbers too small
+  aspect.Scale(100);
```

Comment:
:+1: on calling `Scale()` here but the comment doesn't explain why it's necessary, only that it is necessary.

Maybe something more like
`Must scale aspect higher because setAspectRatio() truncates aspect.width,height to ints`